src/api/challenge.ts

```diff
@@ -98,6 +101,15 @@ export class ChallengeAPIClient {
         if (filter.mint) {
           params.append("mint", filter.mint);
         }
+        if (filter.state) {
+          params.append("state", filter.state);
+        }
+        if (filter.search) {
+          params.append("search", filter.search);
```

Comment:
I saw that but I was thinking it could be nice to be able to combine a search param with another filter. It give a bit more flexibility and it doesn't cost us anything